include/nuttx/net/netconfig.h

```diff
@@ -456,11 +456,11 @@
  * connection requires approximately 30 bytes of memory.
  */
 
-#ifndef CONFIG_NET_TCP_CONNS
+#ifndef CONFIG_NET_TCP_CONNS_PER_BLOCK
 #  ifdef CONFIG_NET_TCP
-#   define CONFIG_NET_TCP_CONNS 10
+#   define CONFIG_NET_TCP_CONNS_PER_BLOCK 10
```

Comment:
The new default in Kconfig is `default 4`  is it better these this match?

Reply:
Done